src/javascript/index.js

```diff
@@ -117,6 +117,7 @@ $(document).ready(function() {
     });
     /////////////////////////////////////////////////////
 
+    /////////////////////////////////////////////////////
```

Comment:
can you revert this, none of the other comments have leading comment bars. It is not necessary for this PR